learning/tour-of-beam/frontend/pubspec.yaml

```diff
@@ -45,8 +45,10 @@ dependencies:
   http: ^0.13.5
   json_annotation: ^4.7.0
   markdown: ^6.0.1
-  playground_components: { path: ../../../playground/frontend/playground_components }
+  playground_components:
+    { path: ../../../playground/frontend/playground_components }
```

Comment:
Revert.

Reply:
Should I change the standard formatting rule because of this?

Reply:
I don't know. I use the auto-formatter in Android Studio, it does not do that.